config/jobs/kubernetes/test-infra/test-infra-presubmits.yaml

```diff
@@ -50,6 +50,8 @@ presubmits:
   # Please keep this in sync with the `ci-test-infra-prow-checkconfig` job
   - name: pull-test-infra-prow-checkconfig
```

Comment:
Please don't modify this job in place. Duplicate this job instead, aka creating a new job called `pull-test-infra-prow-checkconfig-beta` which is optional and validates with unknown-fields-all

Reply:
done!